src/test/java/TestUtils.java

```diff
@@ -0,0 +1,20 @@
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.text.pdf.*;
+import com.scrive.pdftools.tools.utils.PdfReaderAutoclosable;
+import com.scrive.pdftools.tools.utils.PdfStamperAutoclosable;
+
+import java.io.*;
+
+public class TestUtils {
+
+    public static boolean checkDocumentIsFlat(byte[] fileFlattened) throws IOException, DocumentException {
+        boolean isflat;
```

Comment:
no need to create this out-of-scope variable better to just return directly the value or make it local to the try block

Reply:
Yes. Now returns the bool from the setField function directly.